{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -127,6 +127,18 @@ matrix:
           stage: Initial tests
           env: TOXENV=codestyle
 
+        # Run documentation link check in a cron job.
+        - language: python
+          python: 3.7
+          name: Documentation link check
+          stage: Cron tests
+          env: TOXENV=linkcheck
+          addons:
+              apt:
+                  packages:
+                      - graphviz
+                      - pandoc
```

Comment:
why is this a dependency in the template?

Reply:
I'm not sure if they're necessary. I want to do A/B testing to see if travis runs correctly with/out them.
graphviz is a dependency in the example provided by @pllim. In my experience pandoc is necessary for links in jupyter notebooks, when they are included as examples in the docs.

Reply:
Linkcheck does a partial doc build and then scan for the link, I think. But I don't see why the stuff in this template would  need graphviz or pandoc. 🤷

Reply:
The template prime audience is libraries, and those most often don't include jupyter notebooks. So I don't think any of the jupyter dependencies should pulled into the template by default.

Reply:
graphviz is necessary for the inheritance diagrams, so it should be there if the API docs does the diagrams by default, but should be removed if not.

Reply:
I am not sure if linkcheck cares about inheritance diagram.

Reply:
Good point on the prime audience. I'll move the pandoc to be specific to https://github.com/nstarman/project_template.

Reply:
> I am not sure if linkcheck cares about inheritance diagram.

I haven't confirmed this, but I think it's included because the inheritance diagram is meant to be (though often isn't) linked to the docs for the corresponding class.